src/plugins/github/nodes.test.js

```diff
@@ -45,6 +46,10 @@ describe("plugins/github/nodes", () => {
     subtype: "USER",
     login: "decentralion",
   });
+  const commit = (): GitNode.CommitAddress => ({
+    type: GitNode.COMMIT_TYPE,
+    hash: "7ff6fd7b565057b2ff6d9105fd89ae0b372f6c09",
```

Comment:
What is `7ff6fd7b565057b2ff6d9105fd89ae0b372f6c09` a hash of? I don’t
see it in SourceCred or either of our example repositories.

Reply:
Just a random hash I had in my working state. I changed it to 000...000